tests/unit/reaction/conservation_rules/test_clebsch_gordan.py

```diff
@@ -0,0 +1,228 @@
+from typing import List, Tuple
+
+import pytest
+
+from expertsystem.particle import Spin
+from expertsystem.reaction.conservation_rules import (
+    IsoSpinEdgeInput,
+    SpinEdgeInput,
+    SpinNodeInput,
+    clebsch_gordan_helicity_to_canonical,
+    isospin_conservation,
+    spin_conservation,
+)
+
+from .test_spin import __create_two_body_decay_spin_data
```

Comment:
I'm surprised this dunder import works (because of the name mangling).

Reply:
Indeed. The importing must care of this somehow

Reply:
But I thought the point of dunders is to protect them from being accessed from another scope.